### ISSUE TITLE
Closing Fast Server make it turn off not correctly.

### ISSUE THREAD

**Issue by Eniotnacram** [suriyun-production/mmorpg-kit-docs#2684]
Hello,

As per the title, when I click X to turn off the server and X to hard close it, the server still happears in my Task Manager processes. I know on a published game I should wait, but when im doing development and simply want to close it ASAP to build it again, its not really a nice workflow to manually close it in my processes every single time.

Building latest kit, latest unity 6 version, on windows.

**Comment by insthync** [comment]
can't it be clicked again the second time it should be closed

**Comment by Eniotnacram** [comment]
Yes when it's clicked the first time, the countdown starts, on a real production obviously I would wait, but when I'm just testing and simply want to close it and rebuild asap I press the X again to close it right away. But it keep it running in my processes. Before I realized this, i had like 20 servers stacked up in my processes

**Comment by Eniotnacram** [comment]
Want a video?

**Comment by NatS0e** [comment]
Yea it happened to me as well. I opened a topic and said it worked when I tried to restart my PC. So yea i figured it out that there process was still running in Task manager even thougt I ended the server.

**Comment by Eniotnacram** [comment]
Seems like its fixed in 1.93f... Even if i close my server before the time, it doesnt stay in processes anymore.
